commons-statistics-descriptive/src/main/java/org/apache/commons/statistics/descriptive/Variance.java

```diff
@@ -109,6 +109,9 @@ public static Variance create() {
      */
     public static Variance of(double... values) {
         final double mean = Mean.of(values).getAsDouble();
+        if (!Double.isFinite(mean)) {
+            return StorelessSampleVariance.create(Math.abs(mean), mean, values.length, Math.abs(mean));
```

Comment:
Correction
```java
return StorelessSampleVariance.create(Math.abs(mean), mean, values.length, mean);
```
The second value is a proxy for the non-finite sum of the values, so we should not use abs.